lib/ruby_lsp/server.rb

```diff
@@ -690,6 +690,20 @@ def shutdown
 
     sig { params(config_hash: T::Hash[String, T.untyped]).void }
     def perform_initial_indexing(config_hash)
+      index_core_classes
+      index_ruby_code(config_hash)
+    end
+
+    sig { void }
+    def index_core_classes
+      start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      RubyIndexer::RBSIndexer.new(@global_state.index).index_core_classes
+      end_time = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      $stderr.puts("Indexed Ruby core classes in #{(end_time - start_time).round(2)} seconds")
+    end
```

Comment:
We intentionally moved any sort of benchmarking to the client to avoid doing more work on the server. Let's not benchmark the execution time here please.

We can add benchmark scripts if necessary, but I honestly think it should be fine.